core/src/components/description-list/description-list.stories.js

```diff
@@ -0,0 +1,103 @@
+export default {
+  title: "Components/Description List",
+  argTypes: {
+    variant: {
+      options: ["regular", "compact"],
+      control: { type: "radio" },
+    },
+  },
+};
+
+const html = String.raw;
+
+const DL_configuration = {
+  regular: {
+    gridStyles: "row-gap: 1rem; column-gap: 2rem",
+    colon: "",
+    parenthesis_open: "",
+    parenthesis_closed: "",
+    comma: "",
+  },
+  compact: {
+    gridStyles: "row-gap: 0; column-gap: 1.5rem",
+    colon: html`<span class="fudis-dl-item-term__compact__colon"
```

Comment:
I think **colon** and **comma** `span` elements should have `aria-hidden` attributes.